lib/web_ui/lib/src/engine/conic.dart

```diff
@@ -33,7 +33,7 @@ class Conic {
 
     // Split conic into quads, writes quad coordinates into [_pointList] and
     // returns number of quads.
-    assert(subdivideCount > 0);
+    assert(subdivideCount >= 0 && subdivideCount <= _maxSubdivisionCount);
```

Comment:
Isn't `subdivideCount <= _maxSubdivisionCount` guaranteed by the implementation of `_computeSubdivisionCount`?

The documentation of `_computeSubdivisionCount` says: _The number of subdivisions never exceed \_maxSubdivisionCount_

Should the second half of this assert be moved to the _countSubdivisionCount method, so the postcondition in the documentation is "guaranteed"?

```dart
assert(pow2 <= _maxSubdivisionCount);
return pow2;
```

Reply:
Added check here to make sure if implementation breaks, this will throw.